UPGRADE_GUIDE.md

```diff
@@ -138,7 +138,7 @@ active when the index route is active.
 </Route>
 
 // will be active only when home is active, not when about is active
-<IndexLink to="/">Home</Link>
+<Link to="/">Home</Link>
```

Comment:
Fixed.

Justed noticed docs/basics/IndexRoutes.md (which explains IndexLink) is not in the TOC.

Should I add it? Or is this page still a WIP?